### ISSUE TITLE
Slow on new Chrome 

### ISSUE THREAD

**Issue by Bsmoder** [VincentGarreau/particles.js#306]
Please check richieinvest.com on Chrome and Safari. 

Somehow with the new version of Chrome, particles have become extremely slow. It works perfectly on Safari tho, same mac. 

I tried even reinstalling chrome.

Did anyone have the same problem recently?

**Comment by jason-parser** [comment]
@Bsmoder have you found any solution to this problem?

**Comment by amandah** [comment]
I have this issue as well.

**Comment by Bsmoder** [comment]
@jason-parser @amandah Yes, apparently it had nothing to do with particles but with chrome, 

Preferences --> Use hardware acceleration when available --> off

**Comment by peterkarcz** [comment]
I got the same lag issue. @Bsmoder I tried both on and off, issue persists. 
Anyone with a solution yet?

**Comment by hirako2000** [comment]
disable retina_detect

**Comment by alavry** [comment]
disabling retina_detect has no effect towards performance on my end

**Comment by alavry** [comment]
fixed for me by enabling chrome://flags/#ignore-gpu-blacklist - is it thus impossible to implement a solution server-side?

**Comment by EduardoHidalgo** [comment]
@hirako2000 your answer worked perfectly. I has slow movement on mobile devices. Ty. 10/10

**Comment by dngraphisme** [comment]
> disable retina_detect

Hello, yes the same for me :) Thanks

**Comment by Kerry-Jr** [comment]
@alavry . Can you please specify which setting you changed in chrome://flags/#ignore-gpu-blacklist and what you set it too. Thanks.

**Comment by Kerry-Jr** [comment]
alavary, lol nevermind I see it now. Sorry new to this stuff. Thanks again
